packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.html

```diff
@@ -9,14 +9,14 @@
     @keyup.esc="$emit('input', '')"
     @blur="$emit('blur')"/>
   <!-- @slot -->
-  <slot name="icon" v-if="icon">
+  <slot name="icon">
     <svg class="sf-search-bar__icon" width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@DonNicoJs can you change <svg> to `<SfIcon/>`?

Reply:
@przspa Done :)